packages/ext-gen/templates.2/webpack.config.js.tpl.default

```diff
@@ -12,6 +12,7 @@ module.exports = function (env) {
   var browser     = get('browser',     'yes')
   var watch       = get('watch',       'yes')
   var verbose     = get('verbose',     'no')
+  var cmdopts     = get('cmdopts',     '')
```

Comment:
Is the input here verified in some way in the plugin? We need to verify it's valid input and we could do that in the plugin and throw an error if not. @wemersonjanuario

Reply:
@jnsencha this checks is in ext-allshared https://github.com/sencha/ext-allshared/blob/3f457de35e075a7f2a836db760908f7cd889c30b/packages/ext-webpack-plugin/src/pluginUtil.js

_getDefaultOptions method and  _getValidateOptions has an entry for that. Are you refering that?

Reply:
@wemersonjanuario those could be used for this I think. Do you want to implement them? Of course, we are "safe" because Cmd itself will not work and will throw an error if the parameters are not correct. If you'd rather leave it, I'll approve this merge.